package/src/index.ts

```diff
@@ -50,6 +50,41 @@ export class CandleClient {
     this.candle.initialize(appUser);
   }
 
+  public presentTradeExecutionSheet(input: {
+    tradeQuote: TradeQuote;
+    presentationBackground?: PresentationBackground;
+    completion?: (
```

Comment:
#q why not just make this function async rather than use callback pattern? 
```suggestion
public presentTradeExecutionSheet(input: {
     tradeQuote: TradeQuote;
     presentationBackground?: PresentationBackground;
}): ({ kind: "success" } & Trade) | { kind: "failure"; error: string }
```